TutByTests/TutByTests/Tests.cs

```diff
@@ -0,0 +1,37 @@
+using NUnit.Framework;
+using Training.Helpers;
+using Training.Pages;
+
+namespace TutByTests
+{
+    public class Tests
+    {
+        private HomePage _homePage;
+
+
+        [SetUp]
+        public void Setup()
+        {
+            _homePage = new HomePage();
+            _homePage.OpenHomepage();
+            Assert.IsTrue(_homePage.IsPageOpened);
```

Comment:
Please, add the second parameter (it requires string message) for this IsTrue method to get more detailed information in case if test fail on this step.